### PR TITLE
LPD-41878 Missing classes for AssetPublisherPortlet in BundleSiteInitializer

### PR DESCRIPTION
**Pull request by MrJohn1911**
https://liferay.atlassian.net/browse/LPD-41878

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by jonathanmccann** [comment]
ci:test:site-management

**Comment by david-truong** [comment]
@MrJohn1911 do we need a test for this?

**Comment by jonathanmccann** [comment]
ci:test:site-management

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:site-management

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/layouts/4_test-public-layout/page-definition.json [hunk 1]
```diff
@@ -32,6 +32,33 @@
...
+							"anyAssetType": "false",
+							"classNameIds": [
+								"[$CLASS_NAME_ID:com.liferay.blogs.model.BlogsEntry$]",
+								"[$CLASS_NAME_ID:com.liferay.bookmarks.model.BookmarksEntry$]",
+								"[$CLASS_NAME_ID:com.liferay.bookmarks.model.BookmarksFolder$]",
+								"[$CLASS_NAME_ID:com.liferay.calendar.model.Calendar$]",
+								"[$CLASS_NAME_ID:com.liferay.dynamic.data.mapping.model.DDMStructure$]",
+								"[$CLASS_NAME_ID:com.liferay.document.library.kernel.model.DLFolder$]",
+								"[$CLASS_NAME_ID:com.liferay.document.library.kernel.model.DLFileEntry$]",
+								"[$CLASS_NAME_ID:com.liferay.dynamic.data.lists.model.DDLRecord$]",
```
[the remark this answers is not in the document]
Comment (jonathanmccann) [reply modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/layouts/4_test-public-layout/page-definition.json:49]: The three failing site management tests are unrelated and failing everywhere.

**Pull request closed without merging**